.importjs.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  importDevDependencies: true,
+  environments: ['node', 'browser'],
+  declarationKeyword: 'import',
+};
```

Comment:
Ah, cool! I will check it out.

Do you think it could be worthwhile adding a file comment linking to that project? I wasn't able to pull up that project from my google searches (although it very well could be that i'm bad at googling). No worries either way.